thinkhazard/static/less/report.less

```diff
@@ -252,6 +252,21 @@ body {
   }
 }
 
+footer {
+  display: table;
+  white-space: nowrap;
+  &:before, &:after {
+    border-top: 5px solid #e6e6e6;
+    content: '';
+    display: table-cell;
+    position: relative;
+    top:0.5em ;
```

Comment:
If I use this instead of 'top: 0.5em;' the whole bar is moved too far down (see newly committed code) Do I have to add something else as well?